libs/cardano-ledger-api/cardano-ledger-api.cabal

```
@@ -1,6 +1,6 @@
 cabal-version: 3.0
 name: cardano-ledger-api
-version: 1.12.0.0
+version: 1.12.1.0
```

We have release and versioning process that describes when versions need to be bumped. I know that you are not familiar with it, but in the nutshell the breaking  version has already been bumped, so there is no need to bump it any further
```suggestion
version: 1.12.0.0
```